# params/dmw_params.yaml
# Common parameters to the dwm1001 code
---

obj_det: {
    cam_type:                    "/zed/",                                               # the root name of the camera to connect to
    net_file:                    "/home/ros/catkin_ws/src/robot/obj_det/nets/dc_3_v10_20_20_100_Laptop_final_net.dat",     # the location of the network weights file
}
